phpBB/styles/prosilver/theme/jquery.atwho.css

```diff
@@ -0,0 +1,79 @@
+.atwho-view {
+	position: absolute;
+	background: #ffffff;
+	border: 1px solid #dddddd;
+	border-radius: 3px;
+	box-shadow: 0 0 5px rgba(0, 0, 0, 0.1);
+	color: #000000;
+	display: none;
+	left: 0;
+	margin-top: 18px;
+	min-width: 120px;
+	top: 0;
+	z-index: 11110 !important;
+}
+
+.atwho-view .atwho-header {
+	border-bottom: solid 1px #eaeff1;
+	color: #6f8092;
+	cursor: pointer;
+	font-size: 11px;
+	font-weight: bold;
+	margin: 5px;
+	padding: 5px;
+}
+
+.atwho-view .atwho-header .small {
+	color: #6f8092;
+	font-size: 12px;
+	font-weight: normal;
+	float: right;
+	margin-right: -5px;
+	padding-top: 2px;
+}
+
+.atwho-view .atwho-header:hover {
+	cursor: default;
+}
+
+.atwho-view .cur {
+	background: #3366ff;
+	color: #ffffff;
+}
+
+.atwho-view .cur small {
+	color: #ffffff;
+}
+
+.atwho-view strong {
+	color: #3366ff;
+}
+
+.atwho-view .cur strong {
+	font: bold;
+	color: #ffffff;
+}
+
+.atwho-view ul {
+	/* width: 100px; */
+	overflow-y: auto;
+	max-height: 800px;
+	margin: auto;
+	padding: 0;
+	list-style: none;
+}
+
+.atwho-view ul li {
```

Comment:
pls add classes to all html elements and style them via the classes. Do not style the elements directly this is bad specificity leading to problems in future versions down the road